File: src/fib.go
package main
import "fmt"

var num = 45

func fib(n int) int {
	if n <= 2 {
		return 1;
	} else {
		return fib(n - 1) + fib(n - 2)
	}
}

func main() {
	fmt.Println(fib(num))
}

File: src/fib.rs
fn fib(n: i64) -> i64 {
    if n <= 2 {
        1
    } else {
        fib(n - 1) + fib(n - 2)
    }
}

fn main() {
    let num = 45;
    println!("{}", fib(num))
}
